wp-job-manager-functions.php

```diff
@@ -399,7 +400,12 @@ function wp_job_manager_create_account( $args, $deprecated = '' ) {
     }
 
     // Notify
-    wp_new_user_notification( $user_id, $password );
+    if ( version_compare( $wp_version, '4.3.0', '<=' ) ) {
+    	wp_new_user_notification( $user_id, $password );
+    }
+    elseif {
```

Comment:
To nitpick: `elseif` should be on the same line as the `}` above it per WP coding standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style 

Also, not sure why it's `elseif` instead of just `else`?